docs/rules/order.md

```diff
@@ -89,3 +91,56 @@ You can set the options like this:
 ```js
 "import/order": ["error", {"groups": ["index", "sibling", "parent", "internal", "external", "builtin"]}]
 ```
+
+### `newlines-between: [bool]`:
+
+Checks existence of new lines between and inside import groups. For sake of examples let's assume `groups` settings like:
+
+```js
+groups: [
+	['builtin', 'index'],
+	['sibling'],
+	['parent', 'external'],
+],
+```
+
+**a)** Not setting `newlines-between` option will cause rule not to check any empty lines validity.
+
+**b)** With `newlines-between: true` this will be ok:
```

Comment:
Do not set a new group setting for the example, let's keep it simple and keep the default setting.
Instead of creating a section for every possibility, set and show it in the example, like

```js
/* eslint import/order: ["error", {"newlines-between": true}] */
var fs = require('fs');
```

Maybe keep examples short? With like 2 groups and 3 imports (suggestion, do what you think renders best here)
And if you need to have comments, add a space after `// ` for consistency :)